webpack/webpack.config.server.js

```diff
@@ -3,7 +3,13 @@ import webpack from 'webpack';
 import HappyPack from 'happypack';
 
 const root = process.cwd();
-const serverInclude = [path.join(root, 'src', 'server'), path.join(root, 'src', 'universal')];
+const serverInclude = [
+  path.join(root, 'src', 'server'),
+  path.join(root, 'src', 'universal')
+];
+const serverExclude = [
```

Comment:
true true, that reminds me i need to come up with an elegant way to do server development without setting the PRODUCTION flag so we can actually make use of piping...

nuke the `serverExclude` and this PR LGTM!